test/Chainweb/Test/Pact/PactMultiChainTest.hs

```diff
@@ -1408,3 +1426,8 @@ cbResult = do
   (o,_h) <- getPWO chid
   liftIO $
     decodeStrictOrThrow @_ @(CommandResult Hash) (_coinbaseOutput $ _payloadWithOutputsCoinbase o)
+
+checkLocalResult :: Either PactException LocalResult -> (CommandResult Hash -> PactTestM ()) -> PactTestM ()
+checkLocalResult r checkResult = case r of
+  Right (LocalResultLegacy cr) -> checkResult cr
+  res -> liftIO $ assertFailure $ "Expected LocalResultLegacy, but got: " ++ show res
```

Comment:
Optional nit: this is really checking for _legacy_ results, which is not the same as peeking into and handling local results in general. This should probably have a comment and namechange to that effect.